app/models/species.rb

```diff
@@ -1,4 +1,4 @@
-class Specie < ApplicationRecord
+class Species < ApplicationRecord
```

Comment:
Yes [you're right](https://english.stackexchange.com/questions/120068/how-can-i-distinguish-between-the-singular-and-plural-of-species) my bad!